grapher/axis/AxisOptions.ts

```diff
@@ -0,0 +1,99 @@
+import { ScaleType } from "grapher/core/GrapherConstants"
+import { extend } from "lodash"
```

Comment:
🕵️‍♂️🔍

Reply:
Ah!!! Thank you. That might explain bundle check, as net code added should be negative.